gateway/events.go

```diff
@@ -122,18 +122,18 @@ type (
 	// client sends over GuildSubscriptions with the Channels field used.
 	// The State package does not handle this event.
 	GuildMemberListUpdate struct {
-		ID          string            `json:"id"`
-		GuildID     discord.Snowflake `json:"guild_id"`
-		MemberCount uint64            `json:"member_count"`
-		OnlineCount uint64            `json:"online_count"`
+		ID          string          `json:"id"`
+		GuildID     discord.GuildID `json:"guild_id"`
+		MemberCount uint64          `json:"member_count"`
+		OnlineCount uint64          `json:"online_count"`
 
 		// Groups is all the visible role sections.
 		Groups []GuildMemberListGroup `json:"groups"`
 
 		Ops []GuildMemberListOp `json:"ops"`
 	}
 	GuildMemberListGroup struct {
-		ID    string `json:"id"` // either discord.Snowflake Role IDs or "online"
+		ID    string `json:"id"` // either discord.RoleID or "online"
```

Comment:
Original comment had plural IDs, is that correct?

Reply:
Probably not. Singular seems to be valid.